pyvcloud/vcd/vdc.py

```diff
@@ -415,6 +426,14 @@ def update_disk(self, name, size, new_name=None, description=None,
         else:
             disk = self.get_disk(name)
 
+        if storage_profile_name is not None:
+            storage_profile = self.get_storage_profile(storage_profile_name)
+            sp = etree.XML(etree.tostring(storage_profile, pretty_print=True))
+            sp_href = sp.attrib['href']
+            diskParms.append(E.StorageProfile(href=sp_href,
+                                              name=storage_profile_name))
+            print(etree.tostring(diskParms, pretty_print=True))
```

Comment:
Do we need to print the diskParams here? Will it be seen by the vcd-cli user? If so, does he need to see the diskparams?

Reply:
Its not, i will remove it.